lib/core/src/sdk.rs

```diff
@@ -2706,11 +2719,55 @@ impl LiquidSdk {
         Ok(config)
     }
 
+    async fn bip353_parse(&self, input: &str) -> Option<String> {
```

Comment:
Looking good! I think we should move this code into the `sdk_common` crate under the already existing [`parse`](https://github.com/breez/breez-sdk-greenlight/blob/main/libs/sdk-common/src/input_parser.rs#L183) method, along with the Cargo dependencies (dns-resolver)